docs/api/selector.md

```diff
@@ -79,11 +78,11 @@ const myComponents = wrapper.find(MyComponent);
 
 
 
-### 4. A React Component’s displayName
+### 3. A React Component’s displayName
 
 enzyme allows you to find components based on a component’s `displayName`. If a component exists
 in a render tree where its `displayName` is set and has its first character as a capital letter,
```

Comment:
this isn't technically accurate; for example, things like `withSomeHOC(ComponentName)` can also be found in many cases (as noted below); perhaps this sentence needs updating as well?

Reply:
I don't understand what the problem is.  Isn't displayName sortof self-service?